Mixpanel.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = 'Mixpanel'
-  s.version      = '4.0.0'
+  s.version      = '4.0.0.beta.4'
```

Comment:
we want to remove the beta tag, right?

Reply:
will let the script bump the version to 4.0.0 after merge